ioctl/cmd/bc/bcbucket.go

```diff
@@ -86,12 +85,12 @@ func newBucket(bucketpb *iotextypes.VoteBucket) (*bucket, error) {
 		return nil, output.NewError(output.ConvertError, "failed to convert amount into big int", nil)
 	}
 	unstakeStartTimeFormat := "none"
-	unstakeTime, err := ptypes.Timestamp(bucketpb.UnstakeStartTime)
-	if err != nil {
+	if err := bucketpb.UnstakeStartTime.CheckValid(); err != nil {
 		return nil, err
 	}
+	unstakeTime := bucketpb.UnstakeStartTime.AsTime()
 	if unstakeTime != time.Unix(0, 0).UTC() {
-		unstakeStartTimeFormat = ptypes.TimestampString(bucketpb.UnstakeStartTime)
+		unstakeStartTimeFormat = bucketpb.UnstakeStartTime.AsTime().Format(time.RFC3339Nano)
```

Comment:
just `unstakeTime.Format(time.RFC3339Nano)`

Reply:
Like in `ptypes.TimestampString` package https://github.com/golang/protobuf/blob/ae97035608a719c7a1c1c41bed0ae0744bdb0c6f/ptypes/timestamp.go#L81-L87
`ptypes.TimestampString` should be replace with `ts.AsTime().Format(time.RFC3339Nano)` so it can match `unStakeStartTimeFormat`'s type.